src/api/util.js

```diff
@@ -2,6 +2,20 @@
  * Export util functions
  */
 
+const DEG2RAD = Math.PI / 180;
+const EARTH_RADIUS = 6378137;
+const WM_R = EARTH_RADIUS * Math.PI; // Webmercator *radius*: half length Earth's circumference
+const WM_2R = WM_R * 2; // Webmercator coordinate range (Earth's circumference)
+
+// Webmercator projection
+function wmProjection(latLng) {
```

Comment:
function names should be verbs, maybe `project` ?

Reply:
`projectWM`, `wmProject`?